.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,19 @@
+
+name: "Coding Standards"
```

Comment:
I'd rather call this workflow Static analysis as we could stuff in some other tools here later. Example: https://github.com/twirphp/twirp/blob/master/.github/workflows/static.yaml